docs/Makefile

```
@@ -0,0 +1,19 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
```

```suggestion
SPHINXOPTS    = -W
```

This turns all wanrings into errors. Seems pedantic but in my experience without this you end up with a boatload of warnings over time that train you to look away when they scroll past on the screen.